### DISCUSSION TITLE
Update loses all preferences, keybindings and scripts

### DISCUSSION THREAD

**Discussion by sheepdoggie** [freeplane/freeplane#527] [Help] [answered]
I use Freeplane intensively daily in my work and have customized it specifically for my purposes with preferences, keybindings and scripts. But with Flatpak updates occur automatically, which it did going from version1.8.11 to 1.10.3. When that occurred all my customizations were lost, because the user directory is newly created empty. This is a big problem.

**Comment by dpolivaev** [comment, marked as answer]
Freeplane keeps user customization files in separate version dependent directories and copies them only when it goes from 1.7 to 1.8, 1.8 to 1.9, and 1.9 to 1.10 and so on
But it does not delete old files.
If you need to install 1.10 right after 1.8 you can copy all files manually.
